src/MovieModal/MovieModal.jsx

```diff
@@ -0,0 +1,13 @@
+import Modal from "../Modal/Modal";
+import EditForm from "../EditForm/EditForm";
+
+export default function MovieModal({ isOpen, onClose, modalType }) {
+    return (
+        <Modal isOpen={isOpen} onClose={onClose}>
+            <div>
+                {modalType === 'add' ? <h2>add movie</h2> : modalType === 'edit' ? <h2>edit movie</h2> : null}
```

Comment:
It's better to pass the title as a prop.

Reply:
Changed the Modal Component to accept the title as a prop and fewer child elements.